src/include/index/vamana_index.h

```diff
@@ -765,10 +839,10 @@ class vamana_index {
    * the group?
    */
   auto write_index(
-      const tiledb::Context& ctx,
+      tiledb::Context ctx,
```

Comment:
Why change this? We use a const ref in `src/include/index/ivf_flat_index.h` and it seems to work, what is different about this? https://github.com/TileDB-Inc/TileDB-Vector-Search/blob/main/src/include/index/ivf_flat_index.h#L744

Reply:
Also a merge regression.  Big trouble with pass by value on this.  Fixed.